open_api_spec.yml

```diff
@@ -6171,6 +6191,35 @@ paths:
                   headers: *req-id-header
           tags:
               - NFTs
+    /api/v1/nfts/ownership/tokens/spam:
```

Comment:
X-Readme Section is missing here.
It is a section that shows how to use our SDK with this endpoint.
**For example:**

```
x-readme:
        code-samples:
              - language: python
                code: vault_accounts = fireblocks.get_vault_accounts()
                name: Fireblocks SDK Python example
              - language: javascript
                code: const vaultAccounts = await fireblocks.getVaultAccounts();
                name: Fireblocks SDK Javascript example
```

Reply:
Added code samples. The SDK pull requests have not been merged yet, will be merged soon.

Reply:
@lazars14-f thanks